webapp/src/Service/ImportExportService.php

```diff
@@ -595,7 +595,8 @@ protected function importGroupsTsv(array $content, ?string &$message = null): in
             $line = Utils::parseTsvLine(trim($line));
             $groupData[] = [
                 'categoryid' => @$line[0],
-                'name' => @$line[1]
+                'name' => @$line[1],
+                'allow_self_registration' => false,
```

Comment:
@nickygerritsen this is now also needed with your fix.

Reply:
Sure. Seems fine